### DISCUSSION TITLE
Hot to change primary color in dark mode?

### DISCUSSION THREAD

**Discussion by AucT** [twbs/bootstrap#39369] [Q&A]
I tried modify the `_variables-dark.scss` but it doesn't help. I'm getting same output

```
//_variables-dark.scss
// Global colors
//
$primary: #7F00FF !default;
```

**Comment by julien-deramond** [comment]
Right now, this first version of the color modes uses the same primary color for the light and dark modes which is defined by `$primary`. To change it:

```scss
$primary: red;

// Include the remainder of required Bootstrap stylesheets (including any separate color mode stylesheets)
@import "../node_modules/bootstrap/scss/variables";
@import "../node_modules/bootstrap/scss/variables-dark";
```

But the color will be the same in light and dark mode (e.g. the buttons will have the same rendering in both modes). Some future modifications (breaking) will be introduced later on to be able to define two different color palettes for light and dark modes.

Depending on what you're trying to achieve here, the way to define the colors might differ. For instance, here's how to define a different dark rendering for the buttons: https://github.com/orgs/twbs/discussions/38693.
